src/components/TripCard/container.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import XDate from 'xdate';
+import { TripCardUI } from './ui';
+import { TripType } from '../../constants';
+
+export class TripCardContainer extends PureComponent {
+    static propTypes = {
+        trip: TripType
+    };
+
+    static defaultProps = {
+        trip: null
+    };
+
+    formatBookerName() {
+        const { trip } = this.props;
+        return `${trip.bookedBy.firstName} ${trip.bookedBy.lastName}`;
+    }
+
+    formatTravelers() {
+        const { trip } = this.props;
+        const travelerNames = [];
+        trip.travelers.forEach(traveler => {
```

Comment:
in js, we usually use the map array function in cases like this. could you use that instead here? https://www.w3schools.com/jsref/jsref_map.asp#targetText=The%20map()%20method%20creates,for%20array%20elements%20without%20values.